docs/documentation/release_notes/topics/24_0_0.adoc

```diff
@@ -1,3 +1,57 @@
+= Supported user profile and progressive profiling
+
+The user profile preview feature is promoted to be fully supported and user profile is enabled by default!
+
+In the past months, Keycloak team spent huge amount of effort in polishing user
+profile and making it fully supported now. In this release, we continued the effort started in the last release. Lots of improvements, fixes and
+polishing was done based on the thorought testing and feedback from our awesome community.
+
+Few highlights of the user profile feature:
+
```

Comment:
```suggestion
* Fine-grained control over the attributes that users and administrators can manage so that you can prevent unexpected attributes, and values, from being set. 
```